README.md

```diff
@@ -74,10 +74,14 @@ Please be aware that the demos may exhibit significant accessibility issues, suc
   - [Zoom when hover](#zoom-when-hover)
   - [Floating Labels V2](#Floating-labelV2)
   - [Scary animation](#scary-animation)
+  - [ANimated Button](#animated-btn)
+
+  - [Contributors](#contributors)
+  - [Contributing](#contributing)
   - [Light Bulb Animation](#light-bulb-animation)
   - [growing-flower](#growing-flower)
-- [Contributors](#contributors)
-- [Contributing](#contributing)
+  - [Contributors](#contributors)
```

Comment:
@Upendra2003 this portion is twice, fix and push it again. Remove the line no "79 and 80" and add your quick link above the "83 and 84" line.